src/modals/examples/modal-overlay.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.modals.componenet:pfModalOverlay
+ * @element pfModalOverlay
+ * @restrict E
+ *
+ * @description
+ * The Modal Overlay pattern provides a way to quickly load and display important information to the user without navigating
+ * away from the current page. It utilizes the {@link https://angular-ui.github.io/bootstrap/#modal Angular UI modal}
+ * and Patternfly styling. Use cases for Modal Overlays can vary a great deal, but some examples include the following:
+ * <ul>
+ * <li>To remind or prompt users
+ * <li>To load dialogs that require user input before advancing
+ * <li>To load tasks which require a user’s full attention, such as stepping through a wizard flow
+ * <li>To present important information or warnings
+ * </ul>
+ *
+ * @param {boolean} isOpen Flag indicating that the modal should be opened
```

Comment:
I realize this is how aboutModal works, but I keep tripping up on this.  `isOpen` or any `isFooBar` is suppose to just return a boolean True or False; app devs don't typically expect setting a `isWhatever` actually does an action.
Pls change to `showModal` to be clearer of it's usage.  -thanks

Reply:
...and even `showModal` isn't implemented quite correctly as `showModal = false` won't actually hide the modal...although I think that would be easy to implement, however I don't think it's it's a common usecase.